prisma/schema.prisma

```diff
@@ -112,3 +114,14 @@ model ChapterRequest {
   availabilities       String
   questions            String
 }
+ 
+ model Chapter {
+  id                   String    @id @default(auto()) @map("_id") @db.ObjectId
+  chapterName          String
+  location             String
+  dateCreated          DateTime  @default(now())
+  StudentIDs           String[]  @db.ObjectId
+  students             User[] 
```

Comment:
Do we need both `students` and `StudentIDs`? I think we can extract their identifiers from the `User` array.